c2corg_ui/static/js/editing/documentediting.js

```diff
@@ -626,8 +626,11 @@ app.DocumentEditingController.prototype.confirmSave = function(isValid) {
   }
 
   const data = this.scope[this.modelName];
-  const score = this.presetQuality(data);
-  this.storeQuality(data, score);
+  const doctype = app.utils.getDoctype(data['type']);
```

Comment:
I would rather use the condition the opposite way so that a new odcument type would not be included by default.

A shame we cannot use `includes` yet!